projects/aca-content/src/lib/components/search/search-input/search-input.component.scss

```diff
@@ -1,13 +1,33 @@
 $search-width: 594px;
-$search-height: 40px;
+$search-height: 32px;
 $search-background: #f5f6f5;
 $search-border-radius: 4px;
 $top-margin: 12px;
 
 .app-search-container {
   color: var(--theme-foreground-text-color);
+  max-width: 100%;
+  margin: 0 !important;
+
+  .app-search-button {
+    width: 32px !important;
```

Comment:
Do we need all these `important`? You don't override anything from ADF so I think it can be achieved with them